src/proxy.sol

```diff
@@ -14,13 +14,13 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity >=0.5.15 <0.6.0;
```

Comment:
Why are there changes in this file? I would not change this in the PR for moving to foundry.

Reply:
Leaving these versions causes incompatibilities once we've switched to foundry. Not sure why these weren't causing an issue with dapp-tools, but I didn't want to push a PR that left the repo in a broken state, so I included the bare minimum to get contracts building and tests running in this PR. Should I revert or leave these as is?